colour.py

```diff
@@ -39,7 +39,7 @@
 import hashlib
 import re
 import sys
-
+from XKCD_colour_definition import *
```

Comment:
avoir uppercases, and `import *`, in general, I'm not sure this requires to be in another file.

Reply:
I wasn't aware of this good practice, but i'll keep it in mind !
I put it in another file because i wasn't sure adding a thousand line of hard-coded constant would be good for readability, but it can totally be included in the main file !